packages/app/src/layouts/default.vue

```diff
@@ -135,8 +148,9 @@ const showEnableNotificationsBanner = computed(() => {
 
 const query = useQuery({
   query: MainAppQueryDocument,
-  pause: !showHeader.value,
+  pause: isRunMode,
```

Comment:
We do want to pause this, but only during run mode, where we do not use GraphQL.

This wasn't actually as you'd expect before - `pause` wasn't a reactive variable (eg  `computed` or `ref`) so it was getting set to either `true` or `false` and never changing.

I tried passing a reactive variable to get the expected behavior, but this actually breaks things, like [this test](https://app.circleci.com/pipelines/github/cypress-io/cypress/53785/workflows/6e9482bb-f42b-4be9-ad7a-676c7d26c47b/jobs/2219976/tests#failed-test-0). That's because we always want to use GraphQL in open mode (never pausing), even on the runner page, since we use it to hot reload config changes.

For now I corrected the condition:

- use GraphQL during open mode - never pause
- do not use (eg, permanently pause) in open mode

Reply:
This change makes sense to me.  It is the key to getting this query to refetch when it is not showing the header.